kiosk/api/queries/getHero/index.ts

```diff
@@ -0,0 +1,108 @@
+import { fetchGraphQL } from '../..';
+import { HeroResponse, HeroResult, Image } from '../../../interfaces/hero';
+
+export const getHeroes = async (): Promise<{
+  heroes: HeroResult[];
+}> => {
+  let id: string = process.env.NEXT_PUBLIC_CMP_HERO_CONTENT_ID || '';
+  if (id === '') id = 'jnZOQnEVhUyWn1le6m5FyQ';
```

Comment:
Is this debug code that should be removed? If it is removed, change the `id` variable to a const.

Reply:
its not debug. meant to be used as fallback.